tailcfg/tailcfg_view.go

```diff
@@ -173,6 +173,7 @@ func (v NodeView) MachineAuthorized() bool           { return v.ж.MachineAuthor
 func (v NodeView) Capabilities() views.Slice[string] { return views.SliceOf(v.ж.Capabilities) }
 func (v NodeView) ComputedName() string              { return v.ж.ComputedName }
 func (v NodeView) ComputedNameWithHost() string      { return v.ж.ComputedNameWithHost }
+func (v NodeView) DataPlaneAuditLogID() string       { return v.ж.DataPlaneAuditLogID }
```

Comment:
We talked about having two IDs:
* a node-specific audit log ID, and
* a tailnet-specific audit log ID.

Would it make sense to implement both in the same PR so we only have to bump the CapabilityVersion once?

Reply:
Sure. The only reason why I had done it this way so far is because I needed to take care of the former in OSS before I could build things properly in corp, whereas the other one does not have this hard dependency. But it's easy enough for me to make that change at this point.